index.js

```diff
@@ -85,9 +85,9 @@ function bootApp() {
     app.use(require("./src").default)
   }
 
-  server = app.listen(port, () =>
+  server = require('http-shutdown')(app.listen(port, () =>
```

Comment:
FYI Normal imports should just go at the top of the file, require should only be used in special cases (such as hot reloading code).